packages/client/src/transformer/index.ts

```diff
@@ -125,18 +125,17 @@ const toCorrectionValue = (
       }
     }
   } else {
-    if (newSerializedFieldValue)
-      changedValues.push({
-        section: section.id,
-        fieldName: fieldDef.name,
-        newValue: newSerializedFieldValue,
-        oldValue:
-          serializeFieldValue(
-            fieldDef,
-            originalDraftData[section.id][fieldDef.name],
-            originalDraftData[section.id]
-          ) ?? ''
-      })
+    changedValues.push({
```

Comment:
What is the `newSerializedFieldValue` variable and how does removing it affect the flow?

Reply:
`newSerializedFieldValue` is the a updated property of a declaration. Suppose a field had a value which is later updated by the registrar to be an empty field. The check of `if (newSerializedFieldValue)` restricts that from happening.

Reply:
Ah because it is now an empty string? Do we know why the condition was there originally?

Reply:
Not sure but I think we always used to expect the updated value to have a non-empty value.